### ISSUE TITLE
Allow user to change font size

### ISSUE THREAD

**Issue by vecna13** [tigase/siskin-im#35]
**Is your feature request related to a problem? Please describe.**
Currently the user cannot change the font size. This is an accessibility issue, as some users cannot read the small font well or at all.

**Describe the solution you'd like**
Please allow the user to change the font size, either through the app settings or by supporting Dynamic Type.

**Comment by hantu85** [comment]
Yes, you are right. Accessibility features are not supported yet. We will take your suggestion into consideration and most likely include this support in some future version.

**Comment by splacat** [comment]
Yes, please add this feature. Thank you for your great app!

**Comment by vecna13** [comment]
Fixed in 7.0.
